Firebase/InAppMessaging/Analytics/FIRIAMClearcutLogger.m

```diff
@@ -149,6 +149,13 @@ - (void)logAnalyticsEventForType:(FIRIAMAnalyticsLogEventType)eventType
     eventTimeInMs = @((long)nowInMs);
   }
 
+  if (!iid) {
```

Comment:
Hmm I thought this was already in master. https://github.com/firebase/firebase-ios-sdk/blob/master/Firebase/InAppMessaging/Analytics/FIRIAMClearcutLogger.m#L152 not sure why it's showing up here.

Reply:
Looks like a GitHub diff tool bug to me... I guess, it's shown here because branch was created before the change was merged and was cherry picked later on.

Reply:
Yep GitHub diff does not properly handle cherry-pick's.